prow/external-plugins/refresh/main.go

```diff
@@ -84,8 +84,18 @@ func main() {
 	}
 
 	logrus.SetFormatter(&logrus.JSONFormatter{})
-	// TODO: Use global option from the prow config.
-	logrus.SetLevel(logrus.DebugLevel)
```

Comment:
Please use `logrusutil.ComponentInit`. Also, the whole level setting code isn't needed, we do that during config load already: https://github.com/kubernetes/test-infra/blob/8e5725a1d2c4186b7cc5f46cae1a132559a4d0af/prow/config/config.go#L1850

Reply:
Got it. Thanks for your review!